pkg/kube/kubehttp.go

```diff
@@ -53,20 +53,20 @@ func NewKubeHTTPClient() (Kube, error) {
 	).ClientConfig()
 
 	if err != nil {
-		return nil, fmt.Errorf("could not create rest config: %w", err)
+		return nil, nil, fmt.Errorf("could not create rest config: %w", err)
 	}
 
 	scheme := CreateScheme()
 
-	kubeClient, err := client.New(restCfg, client.Options{
+	rawClient, err := client.New(restCfg, client.Options{
```

Comment:
I am curious about this name change. What makes it a raw client? When its used elsewhere its less clear that its a kube client now.

Reply:
Only to further disambiguate it. It could have stayed the same given that the name of this variable is scoped to this function. I didn't want there to be any ambiguity for the reader of this function.